packages/apps/docs/src/Layout.ts

```diff
@@ -16,8 +16,6 @@ export interface ISubHeaderElement {
   title?: string;
   slug?: string;
   children: ISubHeaderElement[];
-  index?: number;
```

Comment:
was needed for hashes of headers.
But this was refactored 2 months ago. not needed anymore